src/common/hook/useQueue/utils/revoke.tsx

```diff
@@ -0,0 +1,27 @@
+import { RevokingJob } from "../../../../types";
+import { utils } from "@govtechsg/open-attestation";
+
+export const getRevokingJobs = async (documents: any[]): Promise<RevokingJob[]> => {
+  return Promise.all(
+    documents.map(async (document) => {
+      let contractAddress = "";
+      contractAddress = utils.getIssuerAddress(document)[0];
```

Comment:
```suggestion
      let contractAddress = utils.getIssuerAddress(document)[0];
```

Reply:
this one will have to refactor in another PR, cos ZL say that the logic to get the contract address should be in another function.